style.css

```diff
@@ -347,3 +354,223 @@ footer {
   font-size: 22px;
   list-style-type: none;
 }
+
+/* ===== Desktop Version ===== */
+
+@media (min-width: 768px) {
+  /* ===== Navbar Section ===== */
+  .navBar {
+    padding: 0 142px 0 130px;
+    align-items: center;
+    height: 72px;
+  }
+
+  .navBtn {
+    display: flex;
+  }
+
+  .navBtn > .fa-bars {
+    display: none;
+  }
+
+  .navList {
+    display: flex;
+    color: #fff;
+    gap: 16px;
+    align-items: center;
+  }
+
+  .navList > li {
+    list-style-type: none;
+    font-size: 16px;
+    font-family: "Inter", sans-serif;
+  }
+
+  /* ===== Section One ===== */
+  .sectionOne {
+    background-image: url("images/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: contain;
+    height: 100vh;
+  }
+
+  .aboutParent {
+    display: flex;
+    flex-direction: row-reverse;
+    gap: 65px;
+    padding: 100px 20px;
+    align-items: center;
+    justify-content: start;
+  }
+
+  .about > h1 {
+    font-size: 60px;
+    font-weight: 400;
+  }
+
+  .about > p {
+    font-size: 30px;
+  }
+
+  .iconDiv {
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .description {
+    padding: 5px 0;
+  }
+
+  .description > p {
+    font-size: 18px;
+    line-height: 26px;
+    width: 556px;
+    height: 123px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  /* ===== Work section ===== */
+
+  .work {
+    display: block;
+    padding: 120px 110px 60px 110px;
+  }
+
+  .midHeading {
+    position: relative;
+  }
+
+  .borderLine {
+    position: absolute;
+    left: 32%;
+    bottom: 35%;
+    width: 63%;
+  }
+
+  .cardOne {
+    display: flex;
+    gap: 24px;
+    padding: 0 110px;
+  }
+
+  .cardOne > img {
+    width: 400px;
+    height: 335px;
+  }
+
+  .imgText > h3 {
+    padding: 0 0 0.5em 0;
+  }
```

Comment:
- [ ] In your design, kindly use the right spacing and sizing of all the elements so that it is the same as the Figma design. Ensure your text wraps like that of the design and the image is the same size too.
### Given design
![Screenshot 2023-08-10 at 20 14 55 (2)](https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/f1e62911-afd9-4d8e-b9f2-91e127c0b8ab)

### Your design
<img width="1200" alt="Screenshot 2023-08-10 at 20 14 55" src="https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/85bb6ff8-ff70-4988-9829-9f688f4c1df3">


- [ ] In this section, the image labelled one in your design is how all the boxes should look when the cursor hovers on it and it should look like the one labelled two when the cursor is removed.

### Given design
![Screenshot 2023-08-10 at 20 15 08 (2)](https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/54479951-9896-4e46-89ce-ded4fc030514)


### Your design

<img width="1119" alt="Screenshot 2023-08-10 at 20 15 08" src="https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/d9b10651-d17d-4377-81ac-85640a951790">